demos/common/python/models/deblurring.py

```diff
@@ -74,20 +74,14 @@ def prepare_outputs(self):
     def preprocess(self, inputs):
         image = inputs
 
-        if image.shape[0] < self.h and image.shape[1] < self.w:
-            # right bottom padding to resize input image to input_layer shape
-            pad_params = {'mode': 'constant',
-                          'constant_values': 0,
-                          'pad_width': ((0, self.h - image.shape[0]), (0, self.w - image.shape[1]), (0, 0))
-                          }
-            resized_image = np.pad(image, **pad_params)
-        else:
-            resized_image = cv2.resize(image, (self.w, self.h))
+        if not (self.h - self.block_size < image.shape[0] <= self.h and self.w - self.block_size < image.shape[1] <= self.w):
+            self.logger.warn("Chosen model size doesn't match image size. The image is resized")
+
+        resized_image = cv2.resize(image, (self.w, self.h))
         resized_image = resized_image.transpose((2, 0, 1))
         resized_image = np.expand_dims(resized_image, 0)
 
-        meta = {'original_shape': image.shape,
-                'resized_shape': resized_image.shape}
+        meta = {'original_shape': image.shape}
         dict_inputs = {self.input_blob_name: resized_image}
         return dict_inputs, meta
```

Comment:
```suggestion
        dict_inputs = {self.input_blob_name: resized_image}
        return dict_inputs, image.shape[1::-1]
```
So you should rename `meta` in `postprocess()` into something like `dsize` or `widthHeight` to make it easier to understand what it is about. And you can pass `dsize` directly into `cv2.resize()`.